server/routerlicious/packages/services-telemetry/README.md

```diff
@@ -67,3 +67,41 @@ const props = {
 
 Lumberjack.log("Sample message", LogLevel.Info, properties);
 ```
+
+### Global Telemetry Context
+
+When using Lumberjack in Node.js, we can take advantage of [Node.js Asynchronous Context](https://nodejs.org/docs/latest-v18.x/api/async_context.html#class-asyncresource) to bind telemetry properties to the async context which ensures their availability in telemetry without manually adding them to a log's properties param.
+
+```typescript
+import {
+	getGlobalTelemetryContext,
+	bindTelemetryContext,
+} from "@fluidframework/server-services-telemetry";
+import { configureLogging } from "@fluidframework/server-services-utils";
+
+// Do this once at server boot.
+// By default, this will enable global context using Node.js AsyncLocalStorage.
+configureLogging(configPath);
+
+// Do this in codepath entrypoints when common telemetry properties become available.
+getGlobalTelemetryContext().bindProperties({ documentId }, () => {
+	// INSIDE OF documentId CONTEXT
+	getGlobalTelemetryContext().bindProperties({ tenantId }, () => {
```

Comment:
I know this is just an example, but just to help me understand it better: wouldn't tenantId usually be in the outer scope in this case?

On second thought, most of the times we get to know both of them at the same time through the request, so maybe my question doesn't even make sense.

Reply:
Yes that's true! This is purely for example sake to show different levels of context using common telemetry properties.